kubeyard/commands/global_commands.py

```diff
@@ -22,7 +22,7 @@ def context(self):
 
 class SetupCommand(GlobalCommand):
     """
-    Set up global context. It should be run after instalation of kubeyard. By default it uses production context.
+    Set up global context. It should be run after instalation of kubeyard. By default it uses development context.
```

Comment:
I don't think there's a default value. If it's not set explicitly, the value is determined by presence of the `minikube` command.

Reply:
Ok, I've added more detailed description.